client/src/components/sideBar/SideBar.js

```diff
@@ -100,15 +108,15 @@ export function SideBar() {
     )
 
     return (
-      <div className='mb-2 w-full rounded-md bg-slate-100/40 p-2'>
+      <div className='mb-2 w-full rounded-md bg-slate-300/40 p-2'>
```

Comment:
Should this be a modal instead? It might be more flexible as one, if we ever add more preferences and the list gets long.

Reply:
well let's park this for now, since the current is functional and deadline is coming up

Reply:
i kinda like it as it is :3, no modal pls

Reply:
if there are more preferences plus more trips, its not gonna work i think

Reply:
If there are more trips overflow with scroll should be fine right?